Documentation/operations/metrics.rst

```diff
@@ -207,6 +207,19 @@ Name                                       Labels
 ``services_events_total``                                                                     Number of services events labeled by action type
 ========================================== ================================================== ========================================================
 
+Cluster health
+~~~~~~~~~~~~~~
+
+========================================== ================================================== ========================================================
+Name                                       Labels                                             Description
+========================================== ================================================== ========================================================
+``unreachable_nodes``                                                                         Number of nodes that cannot be reached
+``unreachable_health_endpoints``                                                              Number of health endpoints that cannot be reached
+``ip_addresses``                           ``family``                                         Number of allocated IP addresses
```

Comment:
Maybe the IPAM section would be a better place for this metric? /cc @joestringer

Reply:
Yes, please put this one under IPAM.